tools/cellprofiler/cellprofiler.xml

```diff
@@ -1,5 +1,5 @@
-<tool id="cp_cellprofiler" name="cellProfiler" version="@CP_VERSION@">
-    <description>runs a cellProfiler pipeline assembled by all the cellProfiler tools upstream in the workflow</description>
+<tool id="cp_cellprofiler" name="CellProfiler" version="@CP_VERSION@">
+    <description>run a CellProfiler pipeline assembled by all the CellProfiler tools upstream in the workflow</description>
```

Comment:
> by all the CellProfiler tools upstream in the workflow

Is that true, if you want to allow also uploading preconfigured pipelines?

Reply:
changed the description